Issues with Xential API update

It seems that this week Xential updated the API and the following issues were seen:

1. In silent mode: the documents were no longer sent to the BPTL callback
2. In interactive mode: building the document through the Xential interface gave this error:
```
Het document kan niet worden opgeslagen. Er is geen opslagpad gedefinieerd.

java.lang.RuntimeException: nl.interactionnext.builder.BuilderException: nl.interactionnext.xutil.exceptions.LocalizedException: There is no storage path defined in the manager.
        at nl.interactionnext.builder.DocumentBuilder.doBuildJob(DocumentBuilder.java:255)
        at nl.interactionnext.builder.BuildJob$BuildJobCallable.call(BuildJob.java:53)
        at nl.interactionnext.builder.BuildJob$BuildJobCallable.call(BuildJob.java:35)
        at nl.interactionnext.xutil.concurrent.ThreadContextUtil.doContextualized(ThreadContextUtil.java:110)
        at nl.interactionnext.xutil.events.ThreadContextAwareCallableWrapper.call(ThreadContextAwareCallableWrapper.java:26)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at nl.interactionnext.xutil.work.v3.Job$1.run(Job.java:63)
        at nl.interactionnext.xutil.work.v3.JobExecutorThread.runJob(JobExecutorThread.java:89)
        at nl.interactionnext.xutil.work.v3.JobExecutorThread.run(JobExecutorThread.java:65)
Caused by: nl.interactionnext.builder.BuilderException: nl.interactionnext.xutil.exceptions.LocalizedException: There is no storage path defined in the manager.
        at nl.interactionnext.builder.AbstractBuilder.buildAndRegisterDocument(AbstractBuilder.java:99)
        at nl.interactionnext.builder.DocumentBuilder.doBuildJob(DocumentBuilder.java:180)
        ... 8 more
Caused by: nl.interactionnext.xutil.exceptions.LocalizedException: There is no storage path defined in the manager.
        at nl.interactionnext.dms.fs.filebaseddms.FileBasedDMSRegistration.getStoragePath(FileBasedDMSRegistration.java:171)
        at nl.interactionnext.dms.fs.filebaseddms.FileBasedDMSRegistration.sendFileToDMS(FileBasedDMSRegistration.java:101)
        at nl.interactionnext.dms.DMSRegistration.addFile(DMSRegistration.java:88)
        at nl.interactionnext.builder.DMSRegistrator.addMimeTypes(DMSRegistrator.java:130)
        at nl.interactionnext.builder.DMSRegistrator.registerDocument(DMSRegistrator.java:83)
        at nl.interactionnext.builder.AbstractDocument.register(AbstractDocument.java:46)
        at nl.interactionnext.builder.AbstractBuilder.buildAndRegisterDocument(AbstractBuilder.java:93)
        ... 9 more
```

--- a/src/bptl/work_units/xential/tasks.py
+++ b/src/bptl/work_units/xential/tasks.py
@@ -1,5 +1,6 @@
 import json
 import uuid
+from time import sleep
 from xml.dom import minidom
 
 from django.conf import settings
@@ -105,7 +106,7 @@ def start_xential_template(task: BaseTask) -> dict:
         "valuesOption": {},
         "attachmentsOption": {},
         "ttlOption": {},
-        "storageOption": {"fixed": True, "target": "NONE"},
+        "storageOption": {"fixed": True, "None": {}},
         "selectionOption": {"templateUuid": template_uuid},
         "webhooksOption": {
             "hooks": [
@@ -154,6 +155,10 @@ def start_xential_template(task: BaseTask) -> dict:
             start_document_url, params={"ticketUuid": ticket_uuid}
         )
         document_uuid = response_data["documentUuid"]
+
+        # It seems that if the buildDocument request is done too soon
+        # after startDocument, the document is not sent to the callback
+        sleep(5)
 
         # Step 3: Build document silently
         # If not all template variables are filled, building the document will not work.

